src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -59,7 +59,20 @@ public void keyboardShortcutsTest() {
         assertEquals(UIComponentFocusEvent.EventType.FILTER_BOX, uiComponentFocusEventType);
         clearUiComponentFocusEventType();
 
+        // jump to first issue using number key(1) or ENTER
+        push(KeyCode.ESCAPE);
+        push(KeyCode.DIGIT1);
```

Comment:
we could do it for single key codes. but then what do you suggest for key combinations? Like <kbd>Ctrl</kbd>+<kbd>Down</kbd>. Also since their are so many of such keys we can fix them in a separate issue no?